### PR TITLE
cleanup slithering, add pipeline job.

### PR DESCRIPTION
**Pull request by pmckelvy1**

### CODE REVIEW

**Comment by pmckelvy1** [comment]
doing a full cleanup and validation of all slither results is a much larger ticket, maybe 13 at least.

this pr adds a job to the pipeline and cleans up some the current HIGH issues raised by slither (verified to be non-issues), as well as a few mediums. it also excludes LOW, INFORMATIONAL, and OBSERVATIONS form the output.

File: contracts/facade/FacadeAct.sol [hunk 1]
```diff
@@ -286,3 +287,4 @@ contract FacadeAct is IFacadeAct, Multicall {
         revert("unrecognized version");
     }
 }
+// slither-disable-end divide-before-multiply
```
Comment (tbrent) [inline contracts/facade/FacadeAct.sol:290]: I think the `divide-before-multiply` was unintentional

**Review by tbrent — APPROVED** [review]
One small nit but other than that LGTM. It's unfortunate there's no detector for implicit up-casts, or something similar. Still, this improves CI.

**Review by tbrent — DISMISSED** [review]

**Review by tbrent — APPROVED** [review]
LGTM

**Pull request merged**